### ISSUE TITLE
'no' incorrectly expanded to number when it comes before '[n of m]'

### ISSUE THREAD

**Issue by 36degrees** [alphagov/reported-bugs#12]
**Upstream bug:** https://github.com/nvaccess/nvda/issues/8255
**Date:** 2018-05-10
**Reported by:** Oliver Byford
**Related to:** https://github.com/alphagov/govuk-frontend/pull/681

---

### Steps to reproduce:
1. Open [this codepen](https://codepen.io/36degrees/pen/PeRNPM) in Firefox (tested in 52.8.0) which includes this code:

```html
<form>
  <fieldset>
    <legend>Have you changed your name?</legend>
    
    <span id="hint">Choose yes or no</span>

    <label>
    <input type="radio" name="changed-name" value="yes" aria-describedby="hint">Yes
    </label>
    
    <label>
    <input type="radio" name="changed-name" value="no" aria-describedby="hint">No
    </label>
  </fieldset>
</form>
```

2. Tab to 'yes' radio button and listen to what NVDA announces

### Expected behavior:
NVDA should announce "Have you changed your name? Grouping, yes radio button not checked, choose yes or **no**, 1 of 2."

### Actual behavior:
NVDA announces "Have you changed your name? Grouping, yes radio button not checked, choose yes or **number** 1 of 2."

### System configuration:
NVDA version:
2017.4

NVDA Installed or portable:
Installed

Other information:
N/A

Windows version:
Windows 10 Version 1709 Build 16299.431

Name and version of other software in use when reproducing the issue:
Firefox 52.8.0

### Other questions:

Does the issue still occur after restarting your PC?
Yes

Have you tried any other versions of NVDA?
No

**Comment by 36degrees** [comment]
Fixed in NVDA 2018 with Firefox 60.2.2esr (at least as of 2018.2, which is what I was able to test in)
